packages/teams-js/test/public/appInitialization.spec.ts

```diff
@@ -1,6 +1,6 @@
 import * as AppHelpersFile from '../../src/internal/appHelpers';
 import * as AppFile from '../../src/public/app';
-import { appInitialization } from '../../src/public/appInitialization';
+import * as appInitialization from '../../src/public/appInitialization';
```

Comment:
could this instead be

`import * as app from '../../src/public/appInitialization'`;

And then you could delete line 13? Not a big deal, but seems like a little bit of indirection that we don't need anymore with your change

Reply:
That wouldn't work with just a one-line code change since appInitialization doesn't have the `_uninitialize` and `_initialize` funtions. I'd have to refactor that as well, which feels out of scope for this work. Did you possibly mean to refer to line 2's import from app.ts?

Reply:
Yes sorry, highlighted the wrong line apparently